docs/providers/amazon/aws/sensors/emr.rst

```diff
@@ -0,0 +1,37 @@
+EMR Sensor Async
+"""""""""""""""""""""""""""""
+
+.. _howto/sensor/EmrContainerSensorAsync:
+
+To waits asynchronously for job running on EMR container to reach the terminal state
+:class:`~astronomer.providers.amazon.aws.sensor.emr.EmrContainerSensorAsync`.
+
+.. exampleinclude:: /../astronomer/providers/amazon/aws/example_dags/example_emr_eks_containers_job.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_sensor_emr_job_container_async]
+    :end-before: [END howto_sensor_emr_job_container_async]
+
+
+.. _howto/sensor:EmrJobFlowSensorAsync:
+
+To waits asynchronously for job running on EMR container to reach the terminal state
```

Comment:
Polls asynchronously for job running on EMR container to reach the terminal state.